w3-hw1-vy/cypress/pages/studentRegistrationForm/confirmDialog.js

```diff
@@ -0,0 +1,80 @@
+export const confirmDialog = {
+    TXT_NAME: "table >tbody >tr:nth-child(1) td:nth-child(2)",
+    TXT_EMAIL: "table >tbody >tr:nth-child(2) td:nth-child(2)",
+    TXT_GENDER: "table >tbody >tr:nth-child(3) td:nth-child(2)",
+    TXT_MOBILE: "table >tbody >tr:nth-child(4) td:nth-child(2)",
+    TXT_DOB: "table >tbody >tr:nth-child(5) td:nth-child(2)",
+    TXT_SUBJECT: "table >tbody >tr:nth-child(6) td:nth-child(2)",
+    TXT_HOBBIES: "table >tbody >tr:nth-child(7) td:nth-child(2)",
+    TXT_PICTURE: "table >tbody >tr:nth-child(8) td:nth-child(2)",
+    TXT_ADDRESS: "table >tbody >tr:nth-child(9) td:nth-child(2)",
+    TXT_STATE_AND_CITY: "table >tbody >tr:nth-child(10) td:nth-child(2)",
+
+    isNameCorrect(firstName, lastName) {
+        firstName = firstName.trimStart().trimEnd();
+        lastName = lastName.trimStart().trimEnd();
+        cy.get(this.TXT_NAME).should("have.text", firstName + " " + lastName);
```

Comment:
Have you tried `TXT_VALUE: "td + td"` then `cy.get(this.TXT_VALUE).contains(value).should("be.visible")`? You can use just one locator to implement all verify because you know how the result should be